services/order.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order
+
+
+@transaction.atomic
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:
+    user = get_user_model().objects.get(username=username)
+
+    order = Order.objects.create(
+        user=user
+    )
+    if date:
+        try:
+            created_at = datetime.strptime(date, "%Y-%m-%d %H:%M:%S")
+            order.created_at = created_at
+        except ValueError:
+            created_at = datetime.strptime(date, "%Y-%m-%d %H:%M")
+            order.created_at = created_at
+
+    order.save()
+
+    for ticket in tickets:
+        Ticket.objects.create(
+            movie_session_id=ticket["movie_session"],
+            order=order,
+            row=ticket["row"],
+            seat=ticket["seat"]
+        )
+
+
+def get_orders(username: str = None) -> QuerySet:

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]